app/javascript/packages/request/index.spec.tsx

```diff
@@ -0,0 +1,120 @@
+import { useSandbox } from '@18f/identity-test-helpers';
+import { request } from '.';
+import '@18f/identity-polyfill';
```

Comment:
> I needed to include this so that I could make a Response object in the node environment.

I'm guessing this is related to lack of implementation in JSDOM (https://github.com/jsdom/jsdom/issues/1724). Could we limit it to `whatwg-fetch` ? Maybe also move to `spec/javascripts/spec_helper.js` in order to consolidate the side-effecty behavior of DOM/globals setup?

Reply:
I might go as far as to suggest avoiding the side-effecty import behavior and instead stub out the global specifically, since we're already doing _some_ handling of `fetch` in `spec/javascripts/spec_helper.js`, and I'd be worried about conflicts.

For example, in that file, we could do...

```ts
import { Response } from 'whatwg-fetch';

// ...

global.Response = Response;
```